packages/scene-composer/src/components/three-fiber/ModelRefComponent/GLTFModelComponent.tsx

```diff
@@ -109,7 +110,11 @@ export const GLTFModelComponent: React.FC<GLTFModelProps> = ({
   const clonedModelScene = useMemo(() => {
     const result = SkeletonUtils.clone(gltf.scene);
     result.traverse((obj) => processObject(component, obj, { maxAnisotropy }));
-
+    if (result.animations.length == 0) {
```

Comment:
Do we only need to copy animations for the top level object or should we add that to the result traverse?

Reply:
I think so gltfs can only have animations in the top level